connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -306,6 +310,35 @@ protected static ConfigDef baseConfigDef() {
                 .withClientSslSupport();
     }
 
+    private String kafkaClusterId;
+
+    public static String lookupKafkaClusterId(WorkerConfig config) {
```

Comment:
We may want to add a comment or even deprecation to this method to prevent people from adding new call sites for it and to note that it should be downgraded in visibility once our testing migration is complete.